applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1230,34 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+=====================
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+Complete the following steps to test upgrading the firmware on both cores at the same time on a gateway device (the procedure is identical for upgrading headset firmware):
+
+1. Install `nRF Connect Device Manager`_ on your Android or iOS device.
+#. Connect an external flash shield to the gateway. See :ref:`nrf53_audio_app_configuration_configure_fota` for details.
+#. Make sure the gateway runs a firmware that supports DFU using external flash memory.
+#. Use the buildprog.py script to create a .zip folder containing new firmware for both cores.
+   This can be done by running the following command:
+
+   .. code-block:: console
+
+      python buildprog.py -c both -b debug -d gateway --pristine -m external
+
+   .. note::
+      If you use Linux, the ``python`` command must excute Python 3.
```

Comment:
Isn't this valid for all buildprog commands in this document, not only the DFU one?

Reply:
This is only a requirement when building for DFU, since the 'python' command is hard coded into some of the DFU Python files.

When building non-DFU things with buildprog.py you can run "python3 buildprog.py (...)", and then the 'python' command can execute Python 2.

Reply:
```suggestion
   .. note::
      If you use FOTA on Linux, the ``python`` command must excute Python 3.
```

Reply:
excute -> execute

Reply:
Changed to When using the FOTA related functionality in the :file:`buildprog.py` script on Linux, it is required that the ``python`` command executes Python 3.

Done